pkg/rke2/psa.go

```diff
@@ -0,0 +1,78 @@
+package rke2
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	defaultPSAConfigFile = "/var/lib/rancher/rke2/server/rke2-pss.yaml"
+)
+
+// setPSAs sets the default PSA's based on the mode that RKE2 is running in. There is either CIS or non
+// CIS mode. For CIS mode, a default PSA configuration with enforcement for restricted will be applied
+// for non CIS mode, a default PSA configuration will be applied that has privileged restriction
+func setPSAs(cisMode bool) error {
+	logrus.Info("Applying Pod Security Admission Configuration")
+	configDir := filepath.Dir(defaultPSAConfigFile)
```

Comment:
Isn't it wrong that `defaultPSAConfigFile` is here hardcoded? I mean, shouldn't we somehow pass config.PodSecurityConfigFile when the user sets the variable `--pod-security-config-file`?

Reply:
I see it now, sorry! stupid comment